.env.sample

```diff
@@ -1,2 +1,4 @@
-testUserName = ''
-testPassword = ""
\ No newline at end of file
+secret_key='hiddenkey'
+testPassword='JESUS'
+conPort='5432'
+testUserName='postgres'
```

Comment:
There needs to be a new line here, there is a plugin to deal with this. This is a week 1 mistake not week 2

Reply:
i have eslint plugin installed, i will correct it